src/extension.ts

```diff
@@ -582,6 +583,10 @@ export async function performIsIgnoredCheck(
   }
 }
 
+function isOpenInEditor(fileUri: string) {
+  return (VSCode.workspace.textDocuments.findIndex(document => document.uri.toString(true) === fileUri) >= 0);
```

Comment:
Minor improvement: I think we could use `some` instead of `findIndex`. Saves a few chars :smile:

Reply:
Good catch, thanks! 👍🏻